drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -753,13 +767,9 @@ void RasterizerStorageGLES3::texture_set_data(RID p_texture, const Ref<Image> &p
 	if (config.keep_original_textures && !(texture->flags & VS::TEXTURE_FLAG_USED_FOR_STREAMING)) {
 		texture->images.write[p_layer] = p_image;
 	}
-#ifndef GLES_OVER_GL
-	if (p_image->is_compressed() && p_image->has_mipmaps() && !p_image->is_size_po2()) {
-		ERR_PRINTS("Texuture '" + texture->path + "' is compressed, has mipmaps but is not of powerf-of-2 size. This does not work on OpenGL ES 3.0.");
```

Comment:
I removed this as AFAIK, we force resizing to power of 2 sizes when compressing to ETC, ETC2 or PVRTC. So S3TC on WebGL was the only situation where this would not happen, and it should now be fixed by this forced decompression.